perf_test/graph/KokkosGraph_triangle.cpp

```diff
@@ -358,7 +358,7 @@ int main (int argc, char ** argv){
 #endif
 
 
-#if defined( KOKKOS_HAVE_CUDA1 )
+#if defined( KOKKOS_ENABLE_CUDA1 )
```

Comment:
What's "CUDA1"?  Did you mean to type `KOKKOS_ENABLE_CUDA` instead?

Reply:
Jup thats a typo we overlooked ...

Reply:
It is not a typo, it is intentional to disable the code.